relay-config/src/config.rs

```diff
@@ -1008,7 +1090,7 @@ pub struct Outcomes {
 impl Default for Outcomes {
     fn default() -> Self {
         Outcomes {
-            emit_outcomes: false,
+            emit_outcomes: EmitOutcomes::None,
```

Comment:
Should we set the default to `EmitOutcomes::AsClientReports`?

Reply:
i think so, in that case we should write some warnings into the changelog imo